docs/get-started/how-it-works.md

```diff
@@ -1,12 +1,9 @@
-# How Does it Work?
-
-## Contrastive metric learning
+# {octicon}`question` How Does it Work?
 
 From an algorithmic perspective,
-**Finetuner** leverages a contrastive metric learning approach to improve your model.
-How does it work?
+Finetuner leverages the contrastive approach to improve your model for similarity matching.
 
```

Comment:
For a getting started page, this page seems a bit boring (not because of the changes but also before). It describes the process of Finetuning relatively detailed but too superficial that a user can understand how to use Finetuner from this. I would suggest to describe some parts like the construction of the training data a bit shorter and rather provide internal links to the respective sections.